packages/dx-react-scheduler/src/plugins/drag-drop-provider.test.tsx

```diff
@@ -417,6 +417,39 @@ describe('DragDropProvider', () => {
       expect(defaultDeps.action.changeAppointment)
         .toBeCalledWith({ change: { startDate: new Date('2018-06-25 00:00'), endDate: new Date('2018-06-26 00:00'), allDay: undefined } }, expect.any(Object), expect.any(Object));
     });
+    it('should call cellIndex with array if allDayElementsMeta is an empty object', () => {
+      const getBoundingClientRect = jest.fn();
+      getBoundingClientRect.mockImplementationOnce(() => ({ height: 20, top: 20, bottom: 40 }));
```

Comment:
We do not re-define this mock so it's implementation can be inlined to getCellRects at line 427